skabase/SKATestDevice/SKATestDevice/SKATestDevice.py

```diff
@@ -197,41 +198,56 @@ def RunGroupCommand(self, argin):
         return argout
         # PROTECTED REGION END #    //  SKATestDevice.RunGroupCommand
 
-    def devLogMsg(self, dev_log_msg, dev_log_level):
+    def dev_logging(self, dev_log_msg, dev_log_level):
         #Element Level Logging
-        if self._element_logging_level >= 1 and dev_log_level == 1:
+        if self._element_logging_level >= int(tango.LogLevel.LOG_FATAL) \
```

Comment:
[pep8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) prefers using parenthesis for wrapping long lines, not `\`.

Reply:
Done